src/Runner.Worker/ExecutionContext.cs

```diff
@@ -550,10 +550,15 @@ public void AddIssue(Issue issue, string logMessage = null)
                 issue.Message = issue.Message[.._maxIssueMessageLength];
             }
 
+            // Tracking the line number (logFileLineNumber) and step number (stepNumber) for each issue that gets created
+            // Actions UI from the run summary page use both values to easily link to an exact locations in logs where annotations originate from
+            if (_record.Order != null)
```

Comment:
do you want to add a basic L0 test to prevent regression?

Reply:
Done! Took way longer that it should have 🤦  Had some trouble with the mocking + setup